### ISSUE TITLE
Issue with "Using the Web Animations API": (short summary here please)

### ISSUE THREAD

**Issue by amateurd** [mdn/content#10318]
MDN URL: https://developer.mozilla.org/en-US/docs/Web/API/Web_Animations_API/Using_the_Web_Animations_API

#### What information was incorrect, unhelpful, or incomplete?
The article itself is good. Unfortunately, the examples are illustrated with GIF animations and don't use the animations API at all as far as I can tell. The presence of the code implies that the animation related to it has been generated using the code shown.

Why it matters? If you search for "Web API animations examples" and find this page it suggests that Web API animations are being shown. It's misleading because if you use a browser which doesn't support them (Web API animatons) they still appear to work. At the very least it should make clear that these are pictures and video clips of something and not actual working examples.

#### Specific section or headline?
Example anmations

#### What did you expect to see?
WebAPI animations

#### Did you test this? If so, how?
Looked at the HTML source

<details>
<summary>MDN Content page report details</summary>

* Folder: `en-us/web/api/web_animations_api/using_the_web_animations_api`
* MDN URL: https://developer.mozilla.org/en-US/docs/Web/API/Web_Animations_API/Using_the_Web_Animations_API
* GitHub URL: https://github.com/mdn/content/blob/main/files/en-us/web/api/web_animations_api/using_the_web_animations_api/index.md
* Last commit: https://github.com/mdn/content/commit/d3c38334923570ada25766f77212b2e808fdebc5
* Document last modified: 2021-10-15T17:04:17.000Z

</details>

**Comment by ninevra** [comment]
If I understand correctly, @wbamberg [suggested](https://github.com/mdn/content/issues/23811#issuecomment-1405379960) moving the code for these examples into mdn/content as [live samples](https://developer.mozilla.org/en-US/docs/MDN/Writing_guidelines/Page_structures/Code_examples#traditional_live_samples).

Am I right in thinking that would require [permission](https://developer.mozilla.org/en-US/docs/MDN/Writing_guidelines/Attrib_copyright_license#if_you_want_to_reuse_or_republish_content) from the author, Rachel Nabors?

**Comment by wbamberg** [comment]
> If I understand correctly, @wbamberg [suggested](https://github.com/mdn/content/issues/23811#issuecomment-1405379960) moving the code for these examples into mdn/content as [live samples](https://developer.mozilla.org/en-US/docs/MDN/Writing_guidelines/Page_structures/Code_examples#traditional_live_samples).
> 
> Am I right in thinking that would require [permission](https://developer.mozilla.org/en-US/docs/MDN/Writing_guidelines/Attrib_copyright_license#if_you_want_to_reuse_or_republish_content) from the author, Rachel Nabors?

I think we should ask, yes. @rachelnabors , we would like, please, to move the examples you wrote for the Web Animations API (https://developer.mozilla.org/en-US/docs/Web/API/Web_Animations_API/Using_the_Web_Animations_API) into mdn/content, rather than linking to codepens. This would make it much easier for us to maintain these examples. The examples on MDN would then be public domain (CC0) per https://github.com/mdn/content/blob/main/LICENSE.md#licenses-for-code-examples-and-snippets . Would you be OK with that? Thank you!

**Comment by teoli2003** [comment]
Note that Mozilla contracted Rachel Nabors to write these examples. cc/ @Rumyra

**Comment by rachelnabors** [comment]
I have to get back to you on this one as I'm a bit slammed at the moment.
What is involved?

**Comment by rachelnabors** [comment]
Thanks! QQ:

1) What are the permissions of the CCO license?
2) Will I be credited in any way for the artwork?
--

**Comment by rachelnabors** [comment]
@Rumyra Just checking in! What are the permissions of the CCO license? And will I get a credit for the artwork in some way? 

Let's make this happen! Hope you are well!

**Comment by hamishwillee** [comment]
@rachelnabors The information about the license are here: https://developer.mozilla.org/en-US/docs/MDN/Writing_guidelines/Attrib_copyright_license#documentation

Pretty much you retain copyright and you don't get specific "mention", but it is possible to track back and determine your contribution if desired.

**Comment by ninevra** [comment]
> @rachelnabors The information about the license are here: https://developer.mozilla.org/en-US/docs/MDN/Writing_guidelines/Attrib_copyright_license#documentation
> 
> Pretty much you retain copyright and you don't get specific "mention", but it is possible to track back and determine your contribution if desired.

Forgive me, but I don't think it's correct that the author of a code sample retains copyright. CC0 is a dedication to the public domain; afaik it's as close as legally possible to giving up copyright entirely. It grants all possible permissions without restriction.

**Comment by ninevra** [comment]
Note: taking a second look, it seems like all the relevant code is currently [MIT licensed](https://codepen.io/rachelnabors/details/PNYGZQ), if that matters. Unclear about the static images.

**Comment by hamishwillee** [comment]
@ninevra Yes CC shares stuff to the public domain, but you retain the rights to do with it what you want, including licensing it elsewhere. You haven't given up copyright, whatever that means.
I'm not a legal expert, and if you have any concerns I suggest you try find one.

**Comment by rachelnabors** [comment]
Sounds good. Port it over
